Update bom - Orca image tag has been updated

girichinna27 opened the pull request:
[OP-20494](https://devopsmx.atlassian.net/browse/OP-20494)
Not able to pause or cancel the running pipeline from non-admin users

girichinna27 committed:
Update bom

https://devopsmx.atlassian.net/browse/OP-20453
Clouddriver image got changed

girichinna27 committed:
Update bom

By mistake had given tag to cloud driver service instead of orca svc. 
cc: Sai Teja

saitejaopsmx reviewed (commented):
@girichinna27 Please update the image in default/service-settings/orca.yml as well.

FYI
@rpavushettiopsmx @rakeshkrmeena

girichinna27 committed:
Update orca.yml

Updated the quay.io/opsmxpublic/ubi8-oes-orca:1.28.4_v8.24.4_4.0.3.1-samlfix image

saitejaopsmx commented:
@TejaswiniOpsmx  Please test this as well

saitejaopsmx reviewed (approved):
LGTM
